offlinevalidator/offlinevalidator.go

```diff
@@ -215,6 +215,12 @@ func parseJSON(templateFile []byte, refTemplate template.Template, logger *logge
 
 	err = json.Unmarshal(templateFile, &refTemplate)
 	if err != nil {
+		syntaxError, ok := err.(*json.SyntaxError)
```

Comment:
`err` could be of type [json.UnmarshalTypeError](https://golang.org/pkg/encoding/json/#UnmarshalTypeError) as well - it also includes offset info.

Reply:
Good point - I'll add this possibility to be detected.